### ISSUE TITLE
Don't assign store to EventEmitter but EventEmitter to Store

### ISSUE THREAD

**Issue by darkyen** [banderson/generator-flux-react#20]
```
var a = assign(EventEmitter.prototype, {
  foo: '3'
});
var b = assign(EventEmitter.prototype, {
 foo: '4'
});

a === b; // true
```

why ? because EventEmitter.prototype is the target, this causes stores to leak into each other.

**Comment by banderson** [comment]
@darkyen your changes have been included in release version 0.3.2 on NPM

**Comment by darkyen** [comment]
Thanks about them, and i love what you did with BaseStore <: very neat !
